client/src/containers/overview/header/parameters/index.tsx

```diff
@@ -104,7 +104,13 @@ export default function ParametersProjects() {
             </SelectTrigger>
             <SelectContent>
               {parameter.options.map((option) => (
-                <SelectItem key={option.value} value={option.value}>
+                <SelectItem
+                  key={option.value}
+                  value={option.value}
+                  disabled={
+                    option.value === PROJECT_PRICE_TYPE.OPEN_BREAK_EVEN_PRICE
```

Comment:
I think it would've been easier, and scalable, adding the disable status in the options definitions instead of here.